recipes/theora/all/conanfile.py

```diff
@@ -1,182 +1,96 @@
-            autotools.make()
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
 
     def package(self):
-        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
-        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
-        if self._is_msvc:
-            include_folder = os.path.join(self._source_subfolder, "include")
-            self.copy(pattern="*.h", dst="include", src=include_folder)
-            self.copy(pattern="*.dll", dst="bin", keep_path=False)
-            self.copy(pattern="*.lib", dst="lib", keep_path=False)
-        else:
-            autotools = self._configure_autotools()
-            autotools.install()
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.la")
-            tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
-            tools.rmdir(os.path.join(self.package_folder, "share"))
+        copy(self, "COPYING", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        copy(self, "LICENSE", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
 
     def package_info(self):
-        self.cpp_info.set_property("pkg_config_name", "theora_full_package") # to avoid conflicts with _theora component
-
-        self.cpp_info.components["_theora"].set_property("pkg_config_name", "theora")
-        prefix = "lib" if self._is_msvc else ""
-        suffix = "_static" if self._is_msvc and not self.options.shared else ""
-        self.cpp_info.components["_theora"].libs = [f"{prefix}theora{suffix}"]
-        self.cpp_info.components["_theora"].requires = ["ogg::ogg"]
-
-        self.cpp_info.components["theoradec"].set_property("pkg_config_name", "theoradec")
-        self.cpp_info.components["theoradec"].requires = ["ogg::ogg"]
-        if self._is_msvc:
-            self.cpp_info.components["theoradec"].requires.append("_theora")
-        else:
-            self.cpp_info.components["theoradec"].libs = ["theoradec"]
-
-        self.cpp_info.components["theoraenc"].set_property("pkg_config_name", "theoraenc")
-        self.cpp_info.components["theoraenc"].requires = ["theoradec", "ogg::ogg"]
-        if self._is_msvc:
-            self.cpp_info.components["theoradec"].requires.append("_theora")
-        else:
-            self.cpp_info.components["theoraenc"].libs = ["theoraenc"]
-
-        # TODO: to remove in conan v2 once pkg_config generator removed
-        self.cpp_info.names["pkg_config"] = "theora_full_package"
+        self.cpp_info.set_property("pkg_config_name", "theora")
+
+        self.cpp_info.components["theora"].libs = ["theora"]
+        self.cpp_info.components["theora"].requires = ["ogg::ogg"]
+
+        # theoraenc and theoradec libraries can be built, but are untested,
+        # so if someonw wants to use them, then uncomment in CMakeLists.txt,
```

Comment:
😭 we should not make recipes have less features.

Reply:
I had forgotten the original recipe had them, I'll put them back in.   The test_package only tested one of the libs though...